assets/src/scripts/components/floodSlider.js

```diff
@@ -0,0 +1,60 @@
+const { select } = require('d3-selection');
+const { createStructuredSelector } = require('reselect');
+
+const { dispatch, link, provide } = require('../lib/redux');
+const { Actions } = require('../store');
+
+const updateSlider = function(node, {stages, gageHeight}) {
+    if (stages.length === 0) {
+        node.property('hidden', true);
+    } else {
+        const heightIndex = stages.indexOf(gageHeight);
+        node.select('input')
+            .attr('min', 0)
+            .attr('max', stages.length - 1)
+            .attr('step', 1)
+            .attr('value', heightIndex)
+            .attr('aria-valuemin', 0)
+            .attr('aria-valuemax', stages.length - 1)
+            .attr('aria-valuenow', heightIndex)
+            .attr('aria-valuetext', gageHeight);
```

Comment:
I wonder if there are ways to provide useful information from a map to a screen reader? Would seem to be difficult..

Reply:
I have been thinking about this too but haven't done any research.